mne_bids/write.py

```diff
@@ -1125,6 +1126,7 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
     if raw.preload is not False:
         raise ValueError('The data should not be preloaded.')
 
+    bids_root = _path_to_str(bids_root)
```

Comment:
you will also need to update docstring of `write_raw_bids`

Reply:
You mean explicitly state that we support `Path` objects now?

Reply:
yep

Reply:
Ok will do!

Reply:
Resolved in d446acd